ci/build_docs.sh

```diff
@@ -19,29 +19,29 @@ rapids-print-env
 rapids-logger "Downloading artifacts from previous jobs"
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
-VERSION_NUMBER="23.08"
 
 rapids-mamba-retry install \
   --channel "${CPP_CHANNEL}" \
   --channel "${PYTHON_CHANNEL}" \
   rmm librmm
 
-# Build CPP docs
-rapids-logger "Build Doxygen docs"
+export RAPIDS_VERSION_NUMBER="23.08"
+export RAPIDS_DOCS_DIR=${RAPIDS_DOCS_DIR:-"${PWD}/documentation"}
+
+rapids-logger "Build CPP docs"
 pushd doxygen
 doxygen Doxyfile
+mkdir -p "${RAPIDS_DOCS_DIR}/librmm/html"
+mv html/* "${RAPIDS_DOCS_DIR}/librmm/html"
 popd
 
-# Build Python docs
 rapids-logger "Build Python docs"
 pushd python/docs
 sphinx-build -b dirhtml . _html
 sphinx-build -b text . _text
+mkdir -p "${RAPIDS_DOCS_DIR}/rmm/{html,txt}"
```

Comment:
```suggestion
mkdir -p "${RAPIDS_DOCS_DIR}/rmm/"{html,txt}
```

We'll need to uncomment the `{}` for this expansion to work properly.